renovate.json

```diff
@@ -1,45 +1,33 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "local>Altinn/renovate-config"
-  ],
-  "schedule": [
-    "* 5-7 * * 1-4"
-  ],
+  "extends": ["local>Altinn/renovate-config"],
+  "schedule": ["* 5-7 * * 1-4"],
   "timezone": "Europe/Oslo",
-  "postUpdateOptions": [
-    "gomodTidy"
-  ],
+  "postUpdateOptions": ["gomodTidy"],
   "packageRules": [
     {
-      "matchManagers": [
-        "terraform",
-        "gomod"
-      ],
+      "matchManagers": ["terraform", "gomod"],
       "groupName": "{{manager}} non-major dependencies",
-      "matchUpdateTypes": [
-        "minor",
-        "patch"
-      ],
+      "matchUpdateTypes": ["minor", "patch"],
       "groupSlug": "{{manager}}-minor-patch",
-      "matchPackageNames": [
-        "*"
-      ]
+      "matchPackageNames": ["*"]
     }
   ],
   "customManagers": [
-      {
-        "customType": "regex",
-        "managerFilePatterns": [
-          "/(^|/)(.*?/)*.*?\\.ya?ml$/",
-          "/(^|/)(.*?/)*.*?\\.sh$/",
-          "/(^|/)(.*?/)*Dockerfile$/"
-        ],
-        "matchStrings": [
-          "# renovate: datasource=(?<datasource>[a-z-]+?)(?: depName=(?<depName>.+?))? packageName=(?<packageName>.+?)(?: versioning=(?<versioning>[a-z-]+?))?\\s+?default: (?<currentValue>.+?)\\s",
-          "# renovate: datasource=(?<datasource>[a-z-]+?)(?: depName=(?<depName>.+?))? packageName=(?<packageName>.+?)(?: versioning=(?<versioning>[a-z-]+?))?(?: extractVersion=(?<extractVersion>.+?))?\\s+?.+?_VERSION\\s?=\\s?\"?v?(?<currentValue>.+?)\"?(\\s|$)"
-        ],
-        "versioningTemplate": "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}"
-      }
-    ]
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/(^|/)(.*?/)*.*?\\.ya?ml$/",
+        "/(^|/)(.*?/)*.*?\\.sh$/",
+        "/(^|/)(.*?/)*Dockerfile$/"
+      ],
+      "matchStrings": [
+        "# renovate: datasource=(?<datasource>[a-z-]+?)(?: depName=(?<depName>.+?))? packageName=(?<packageName>.+?)(?: versioning=(?<versioning>[a-z-]+?))?\\s+?default: (?<currentValue>.+?)\\s",
+        "# renovate: datasource=(?<datasource>[a-z-]+?)(?: depName=(?<depName>.+?))? packageName=(?<packageName>.+?)(?: versioning=(?<versioning>[a-z-]+?))?(?: extractVersion=(?<extractVersion>.+?))?\\s+?.+?_VERSION\\s?=\\s?\"?v?(?<currentValue>.+?)\"?(\\s|$)",
+        "# renovate: datasource=(?<datasource>helm) registryUrl=(?<registryUrl>(?:https://|oci://)[^\\s]+) packageName=(?<packageName>.+?)\\s+?version:\\s['\"]?(?<currentValue>[^\\s'\"#]+)['\"]?",
+        "version:\\s*['\"]?(?<currentValue>[0-9A-Za-z._+-]+)['\"]?\\s*#\\s*renovate:\\s*datasource=(?<datasource>helm)\\s+registryUrl=(?<registryUrl>(?:https://|oci://)[^\\s]+)\\s+packageName=(?<packageName>[^\\s]+)"
```

Comment:
Is there a need to support both preceding-line and inline?
Might be better to just stick with one if there is nothing forcing ut so support both?

Reply:
sorry, I might have been unclear 🙈 I would prefer to only support preceding-line as that is what we are doing in other places, is there a need for same-line here?